packages/doenetml-worker-rust/lib-doenetml-core/tests/by_component/text_input/with_references.rs

```diff
@@ -40,6 +40,47 @@ fn text_input_reference_child_is_changed_when_update_value() {
     assert_eq!(get_text_value(text_idx, &mut core), "bye");
 }
 
+/// A text component referenced as a child of a text input
+/// will be changed with `UpdateValue`.
+/// This test is checking to make sure the text is changed by the actions
+/// even if it starts blank, i.e., with its default value.
```

Comment:
I would love for this comment to include
```
For example in `<textInput>$t</textInput><text name="t" />`, 
the text input's value can still change even though it has 
a child consisting of an empty text.
```

Reply:
The key point of this test isn't exactly that. It is the motivation for the additional parameter in the props, below. Without that parameter, the text input would still change, but it wouldn't recognize that it had a text child, and so the text child would still stay blank as the text input changes.

I added more text to elaborate further.